tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+	"compileOnSave": true,
+	"compilerOptions": {
+		"allowSyntheticDefaultImports": true,
+		"allowUnreachableCode": false,
+		"alwaysStrict": true,
+		"declaration": true,
+		"declarationMap": true,
+		"emitDecoratorMetadata": true,
+		"esModuleInterop": true,
+		"experimentalDecorators": true,
+		"forceConsistentCasingInFileNames": true,
+		"lib": ["es2023"],
+		// TODO: can't change these to Node16 until
+		// https://github.com/kulshekhar/ts-jest/issues/4198 is resolved
```

Comment:
I'm surprised tsconfig.json's support comments

Reply:
Almost all "microsoft" json formats are canonically actually "jsonc". VSCode & Rush config files are the same. Makes writing helper tooling oh so much fun